src/NotionExporter.ts

```diff
@@ -21,13 +23,15 @@ export class NotionExporter {
    * @param tokenV2 – the Notion `token_v2` Cookie value
    * @param fileToken – the Notion `file_token` Cookie value
    */
-  constructor(tokenV2: string, fileToken: string) {
+  constructor(tokenV2: string, fileToken: string, noFiles: boolean, recursive: boolean = false) {
```

Comment:
I was actually more thinking about something like a config object here:

```suggestion
  constructor(tokenV2: string, fileToken: string, config?: Config = defaults) {
```
see top comment for `Config`

Reply:
Yep, good idea, clearer and cleaner, go ahead. I didn't push a lot of changes for it but that's 👌 . If you don't want/have time, drop a comment here and I'll go for it 👍